src/Attributes.php

```diff
@@ -365,29 +368,19 @@ public function setPrefix($prefix)
     /**
      * Register callback for an attribute
      *
-     * @param string   $name            Name of the attribute to register the callback for
-     * @param callable $callback        Callback to call when retrieving the attribute
-     * @param callable $setterCallback  Callback to call when setting the attribute
+     * @param string        $name           Name of the attribute to register the callback for
+     * @param ?callable     $callback       Callback to call when retrieving the attribute
+     * @param ?callable     $setterCallback Callback to call when setting the attribute
```

Comment:
The whitespace changes are now not necessary anymore. Restore it to this:

```suggestion
     * @param string    $name           Name of the attribute to register the callback for
     * @param ?callable $callback       Callback to call when retrieving the attribute
     * @param ?callable $setterCallback Callback to call when setting the attribute
```